File: 2037. Minimum Number of Moves to Seat Everyone/solution.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
 var minMovesToSeat = function(seats, students) {
  let sortedSeats = [...seats];
  sortedSeats.sort(function(a, b){return a-b});
  let sortedStudents = [...students];
  sortedStudents.sort(function(a, b){return a-b});
  
  let result = 0;
    
  for (let index = 0; index < seats.length; index++){
    let movements = Math.abs(sortedSeats[index] - sortedStudents[index]);
    result += movements;
  }

  return result;
};

File: 2037. Minimum Number of Moves to Seat Everyone/solution.py
class Solution(object):
  def minMovesToSeat(self, seats, students):
    """
    :type seats: List[int]
    :type students: List[int]
    :rtype: int
    """
    sortedSeats = sorted(seats)
    sortedStudents = sorted(students)
    
    result = 0
    
    for seat, student in zip(sortedSeats, sortedStudents):
      movements = abs(seat - student)
      result += movements
      
    return result